.github/quarkus-github-bot.yml

```diff
@@ -226,6 +226,10 @@ triage:
         - extensions/security/
         - extensions/elytron
         - integration-tests/elytron
+    - id: bearer-token
+      labels: [area/security]
+      title: "Bearer token"
```

Comment:
I did a quick search and I think we should go with the following. It won't catch too many false positives AFAICS.

```suggestion
      title: "bearer"
```

Reply:
Yeah you are right, it does look good indeed. Let's get @sberyozkin blessing first and then I'll change it.

Reply:
I changed this, do you agree @sberyozkin ?

Reply:
Thanks @geoand for the PR and @gsmet for the suggestion, yeah, `bearer` alone should be sufficient, we have `bearer authorization`, `bearer authenticaton` in a few places, `token` is not always featuring and is implied in those cases

Reply:
Cool, PR should be ready then